gdcdictionary/schemas/justice_involvement.yaml

```diff
@@ -39,7 +39,46 @@ uniqueKeys:
 
 properties:
   $ref: "_definitions.yaml#/ubiquitous_properties"
-    
+
+  # integer --> categorical for viewing in explorer page
+  J1_category:
+    description: >
+      Categorical of variable for purposes of explorer display
+    type: string
+  
+  J1a_category:
+    description: >
+      Categorical of variable for purposes of explorer display
+    type: string
+
+
+  J2_category:
+    description: >
+      Categorical of variable for purposes of explorer display
+    type: string
+
+  J2a_category:
```

Comment:
i don't see `J2a` and `J2b` in the properties

Reply:
Ah - again thanks for pointing this out - it may be that the previous time this was filled out, an older version of the core measures were used. That is, for the sub-questions -- letters instead of numbers. Given the letters is the new designation, lets keep this and I will change others to letters in subsequent PR.